Include/internal/pycore_traceback.h

```diff
@@ -8,6 +8,9 @@ extern "C" {
 #  error "this header requires Py_BUILD_CORE define"
 #endif
 
+/* Maximum number of error tracebacks to print */
+#define PyTraceBack_LIMIT 1000
```

Comment:
Do we want a macro or do we want a static constant? Also, I think `Py_TRACEBACK_LIMIT` would also be an alternative (it feels a bit weird to have multiple mixed case even if the macro was named like that beforehand).

Reply:
This is the name it was defined with. I would have preferred `PY_TRACEBACK_LIMIT` since it is a constant, but I wanted to keep it as it is because I am not sure if this is the name that will be used for some future resolution.